.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,73 @@
+name: NPM CI
+on:
+  push:
+    paths-ignore:
+      - "docs/**"
+  pull_request:
+    paths-ignore:
+      - "docs/**"
+
+jobs:
+  npm-ci-job:
+    name: "YARN & NPM Build Test"
+  # runs-on: ${{ matrix.os }}
+  # strategy:
+  #   matrix:
+  #     os: [ubuntu-18.04, windows-latest]
+    runs-on: ubuntu-18.04
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10.16'     
+      - name: Restore cached NODE modules
+        uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: v4-dependencies-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            v4-dependencies-${{ hashFiles('yarn.lock') }}
+            v4-dependencies- 
+      - name: "YARN install"
+        run: yarn install --frozen-lockfile
+      - name: "NPM run build"
+        run: npm run build && npm run build:types
+      - name: "Code linting"
+        run: npm run lint
+      - name: "Tests - NODE"
+        run: npm run test:node
+      - name: "Tests - headless browser"
+        run: npm run test:browser
+      - name: "Tests - BrowserStack"
```

Comment:
I think the idea was to 
- run github actions only in the headless browser e.g. use it as the "fast lane" to get fast feedback
- run circleci with browserstack e.g. as the "slow lane" which gives "better" feedback but it takes longer

so we could remove the browserstack here right?